tools/vagrant/vagrant-libvirt.md

```diff
@@ -41,6 +41,14 @@ end
 
 Read more on the `vagrant-libvirt` configuration in the [upstream documentation](https://github.com/vagrant-libvirt/vagrant-libvirt).
 
+
+## Manage guest networks via Vagrant
+
+In Fedora `qemu://session` is used by default. This allows causes vagrant to run in userspace, not requiring password and being much safer. Downside is that not all Vagrant features are accessible, as they require root. To revert for your selected boxes to `qemu://system`:
+
+- Fedora change: https://fedoraproject.org/wiki/Changes/Vagrant_2.2_with_QEMU_Session
+- Upstream documentation: https://github.com/vagrant-libvirt/vagrant-libvirt#qemu-session-support
```

Comment:
These links are not clickable. Please use the `[name](link)` markdown format so that the links become clickable.